smartmin/views.py

```diff
@@ -97,34 +95,7 @@ def has_permission(self, request, *args, **kwargs):
         if not getattr(self, 'permission', None):
             return True
         else:
-            # first check our anonymous permissions
-            real_anon = get_anonymous_user()
-            has_perm = real_anon.has_perm(self.permission)
```

Comment:
The idea is that then all permissions are controlled in a single place, the configuration, as opposed to the views. But not a big deal and worth it to get rid of guardian.